File: php-poo/aulas/aula10/Professor.php
<?php
require_once 'Pessoa.php';
class Professor extends Pessoa
{
    // Atributes
    private $especialidade;
    private $salario;

    // Method
    public function receberAumento($aumento) {
        $this->salario += $aumento;
    }

    // Special Methods - Getters and Setters
    public function getEspecialidade()
    {
        return $this->especialidade;
    }

    public function setEspecialidade($especialidade): void
    {
        $this->especialidade = $especialidade;
    }

    public function getSalario()
    {
        return $this->salario;
    }

    public function setSalario($salario): void
    {
        $this->salario = $salario;
    }

}

File: java-poo/projects/poo-aula11/src/aula11/Professor.java
package aula11;

public class Professor extends Pessoa {
    // Attributes
    private String especialidade;
    private float salario;

    // Methods
    public void receberAumento(float aumento) {
        this.salario += aumento;
    }

    // Special Methods - Getters and Setters
    public String getEspecialidade() {
        return especialidade;
    }

    public void setEspecialidade(String especialidade) {
        this.especialidade = especialidade;
    }

    public float getSalario() {
        return salario;
    }

    public void setSalario(float salario) {
        this.salario = salario;
    }
}
